services/web/server/src/simcore_service_webserver/projects/projects_service.py

```diff
@@ -1548,11 +1553,10 @@ async def get_project_states_for_user(
     user_id: int, project_uuid: str, app: web.Application
 ) -> ProjectState:
     # for templates: the project is never locked and never opened. also the running state is always unknown
-    lock_state = ProjectLocked(value=False, status=ProjectStatus.CLOSED)
```

Comment:
is this OK? or was tit deleted on accident?

Reply:
yes, its ok. That assignment was not used. mypy tole me so :-)